src/DiagonalHessianApproximation.jl

```diff
@@ -13,6 +13,10 @@ and
 Andrei, N. 
 A diagonal quasi-Newton updating method for unconstrained optimization. 
 https://doi.org/10.1007/s11075-018-0562-7
+
+and
```

Comment:
Please improve the documentation here

Reply:
In fact, they don't have any documentation yet because they are new (if you're talking about DBFGS and Wolk).
Do I need to add their formulas in the documentation ?

Reply:
You can do something similar to the other diagonal QN operators

Reply:
@MohamedLaghdafHABIBOULLAH Please be more explicit about DBFGS and Wolk. I presume they will appear in our paper, so you can already cite that. It would be good to summarize each with a sentence or two.